harness/determined/estimator/_estimator_trial.py

```diff
@@ -277,12 +278,25 @@ def control_loop(self) -> None:
                 # re-enters the train_and_evaluate() loop.
                 break
             elif wkld.kind == workload.Workload.Kind.COMPUTE_VALIDATION_METRICS:
-                response_func(
-                    det.util.wrap_metrics(
-                        self._compute_validation_metrics(),
-                        self.estimator_trial_controller.context.get_stop_requested(),
+                try:
+                    response_func(
+                        det.util.wrap_metrics(
+                            self._compute_validation_metrics(),
+                            self.estimator_trial_controller.context.get_stop_requested(),
+                            False,
```

Comment:
Could you use keyword argument format `metrics=False` for these?  Otherwise it is not as clear what `False` means to the reader of the code.